### PR TITLE
Add WAN Replication to README

### PR DESCRIPTION
**Pull request by leszko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ objects:
   data:
     hazelcast.xml: |-
       <?xml version="1.0" encoding="UTF-8"?>
-      <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.10.xsd"
+      <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.12.xsd"
```
Comment (mesutcelik) [inline README.md]: I just do not use versions then xml is validated by the latest xsd in the jar.

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ objects:
   data:
     hazelcast.xml: |-
```
Comment (mesutcelik) [inline README.md]: What about removing inlined xml and use directly yaml support?

Comment (leszko) [reply README.md]: We need to wait for the 3.12 release. Then, I'll update Helm Charts and all README.

File: README.md (hunk 1 above)
Comment (leszko) [reply README.md]: 3.11 for now. Let's change it to 3.12 when 3.12 is released.

**Pull request merged**